src/tests/cmake/CMakeLists.txt

```diff
@@ -0,0 +1,128 @@
+cmake_minimum_required(VERSION 3.24)
```

Comment:
This whole dir should still move to somewhere outside `src/tests`

Reply:
Moving it to `src/scripts/ci` as it isn't really an example anymore either and the usage with CMake's `find_package()` is canonical and doesn't really require an example IMO.